validator/sawtooth_validator/journal/block_validator.py

```diff
@@ -488,10 +493,37 @@ def _get_batch_commit_changes(new_chain, cur_chain):
     def submit_blocks_for_verification(
         self, blocks, consensus, callback
     ):
+
         for block in blocks:
+            LOGGER.debug(
+                "Adding block %s for processing", block.identifier[:6])
```

Comment:
Identifiers should be logged in full (unless this message is going to be logged frequently, in which case it should be dropped or replaced with a metric).